android/app/src/main/java/org/fossasia/openevent/activities/SettingsActivity.java

```diff
@@ -169,6 +176,22 @@ private void setToolbar() {
             getSupportActionBar().setElevation(Utils.dpToPx(4));
```

Comment:
Why setting the elevation dynamically? I think we could use AppBarLayout. It's fine anyways.  

-- android.support.design.widget.AppBarLayout

Then toolbar inside it.